package.json

```diff
@@ -22,12 +22,12 @@
     "build:compile": "gulp compile",
     "build:sassdoc": "sassdoc --config sassdoc.config.yaml ./src/govuk",
     "build:jsdoc": "jsdoc --configure jsdoc.config.js ./src/govuk",
-    "build:package": "gulp build:package --destination 'package' && npm run test:build:package",
-    "build:dist": "gulp build:dist --destination 'dist' && npm run test:build:dist",
+    "build:package": "gulp build:package --destination \"package\"",
+    "build:dist": "gulp build:dist --destination \"dist\"",
+    "postbuild:package": "jest --color --selectProjects \"Gulp tasks\" --testMatch \"**/*build-package*\"",
```

Comment:
By stitching npm commands together with `&&` we saw difficulty detecting "exits"
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference

Reply:
This also allows us to run npm scripts _without_ other scripts (like tests) also running

### Build + test
Includes npm `pre` and `post` scripts

```console
npm run build:compile
npm run build:package
npm run build:dist
```

### Build only (no tests)
Excludes npm `pre` and `post` scripts

```console
npm run build:compile
npm run build:package --ignore-scripts
npm run build:dist --ignore-scripts
```